types/aws-lambda/trigger/lambda-function-url.d.ts

```diff
@@ -0,0 +1,13 @@
+import { APIGatewayProxyHandlerV2, APIGatewayProxyHandlerV2WithIAMAuthorizer } from "./api-gateway-proxy";
+
+/**
```

Comment:
Can we get Event, Result, and Callback aliases added as well? I think many people use those types directly, not just the Handler types.

Reply:
> Can we add Event, Result, and Callback aliases added as well? I think many people use those types directly, not just the Handler types.

Hi @joeykilpatrick , thanks for your feedback, I have removed the duplicated tests as suggested in your other comment.  On this point, could I clarify if you also mean that we should implement `LambdaFunctionURLHandler` separately (i.e. not as an alias of `APIGatewayProxyHandlerV2`) as a result of this suggestion?  Because if we are to alias the Event, Result, and Callback interfaces, then I think it would be weird to _not_ do that since would mean `LambdaFunctionURLHandler` is still essentially `Handler<APIGatewayProxyEventV2, APIGatewayProxyResultV2<T>>`, which would be inconsistent with those interfaces.

Just to be clear, I think it's a good suggestion, but I think it will ultimately involve more interfaces/types that those three if you inspect the interface/type hierarchy of the API Gateway types, unless there's some TS wizardry that I'm overlooking 🙂

Reply:
This is what I mean:
```typescript
export type LambdaFunctionURLEvent = APIGatewayProxyEventV2;
export type LambdaFunctionURLResult<T = never> = APIGatewayProxyResultV2<T>;
export type LambdaFunctionURLCallback = APIGatewayProxyCallbackV2;
export type LambdaFunctionURLHandler<T = never> = APIGatewayProxyHandlerV2<T>
```

Reply:
> This is what I mean:
> 
> ```ts
> export type LambdaFunctionURLEvent = APIGatewayProxyEventV2;
> export type LambdaFunctionURLResult<T = never> = APIGatewayProxyResultV2<T>;
> export type LambdaFunctionURLCallback = APIGatewayProxyCallbackV2;
> export type LambdaFunctionURLHandler<T = never> = APIGatewayProxyHandlerV2<T>
> ```

Ah ok, I wondered about the consistency of the types below the handlers in that case but I'm struggling to communicate that effectively.  Hopefully my most recent commit gets across what I'm trying to explain here.

Reply:
Changes look good. Once the pipeline is fixed, I will approve.

Reply:
Great, thanks for your feedback! 🚀